app/uk/gov/hmrc/gform/gformbackend/GformConnector.scala

```diff
@@ -250,7 +252,7 @@ class GformConnector(ws: WSHttp, baseUrl: String) {
     ws.POST[String, Submission](
       baseUrl + s"/forms/${formId.value}/${formTemplateId.value}/${envelopeId.value}/$noOfAttachments/createSubmission",
       "",
-      Seq(("customerId", customerId))
+      Seq(("customerId", Base64.encodeBase64String(customerId.getBytes(StandardCharsets.UTF_8))))
```

Comment:
Would viable option be to just html escape `customerId`? https://www.freeformatter.com/html-entities.html